corehq/toggles/__init__.py

```diff
@@ -2528,6 +2528,13 @@ def _commtrackify(domain_name, toggle_is_enabled):
     namespaces=[NAMESPACE_DOMAIN]
 )
 
+WEB_USER_INVITE_ADDITIONAL_FIELDS = StaticToggle(
+    'web_user_invite_additional_fields',
+    'Enable additional fields in web user invite form for enhanced user details',
```

Comment:
Should this have a "USH:" in front or is the automatically GAed?

Reply:
Yes it should [63273ea](https://github.com/dimagi/commcare-hq/pull/35124/commits/63273eab0379d42d39031a9a517efeb244ba4bf1)